sdk/js/src/Client.ts

```diff
@@ -387,6 +399,23 @@ export class DVCClient implements Client {
 
         document.addEventListener?.('visibilitychange', this.pageVisibilityHandler)
     }
+
+    private cacheConfig(config: BucketedUserConfig, user: DVCPopulatedUser) {
+        this.store.saveConfig(config, user)
+        this.isConfigCached = false
+    }
+
+    private getConfigCache() {
+        if (this.options.configCacheTTL) {
```

Comment:
`options.configCacheTTL` should be set by default, this should be skipped if `options.disableConfigCache` is set to true

Reply:
it's skipped outside `getConfigCache()` and I've created another ticket to handle the skip config reasons
```
 if (!this.options.disableConfigCache) {
       this.getConfigCache()
} else {
       this.logger.info("Skipping config cache")
}
```

`options.configCacheTTL` is set through caz typescript does not allow ? operator and default value at the same time, not sure what's the best way to set a default value for `configCacheTTL`
```
if (!this.options.configCacheTTL) {
       this.options.configCacheTTL = 604800000
}
```